src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -577,6 +577,8 @@ bool layout_optimizer::convolution_byxf_opt(const layout& input_layout,
             return false;
     }
 
+    bool disable_winograd_conv = node.get_program().get_config().get_property(ov::intel_gpu::disable_winograd_convolution);
```

Comment:
nit: what about checking this condition from `should_use_winograd_2x3_s1` function? I think that will look more straightforward.

Reply:
Applied